base/expr.jl

```diff
@@ -401,13 +401,14 @@ macro constprop(setting)
 end
 
 function constprop_setting(@nospecialize setting)
+    s = setting
     isa(setting, QuoteNode) && (setting = setting.value)
     if setting === :aggressive
         return :aggressive_constprop
     elseif setting === :none
         return :no_constprop
     end
-    throw(ArgumentError(LazyString("@constprop "), setting, "not supported"))
+    throw(ArgumentError("`@constprop $s` not supported"))
```

Comment:
```suggestion
    throw(ArgumentError(LazyString("`Base.@constprop ", setting, "` not supported")))
```
Best to keep using `LazyString` to have good effects for the macro expansion of `@constprop`.

Reply:
Why did you change the use of `s` to a use of `setting`? This would result in an error message like this:

```none
`Base.@constprop unknown` not supported
```

..., so without the `:`.

Reply:
Just a mistake. `setting` would be better.